tests/network/hotplug.go

```diff
@@ -238,6 +238,28 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
+
+		DescribeTable("VMI status should report the MAC address specified in the VM template spec", func(plugMethod hotplugMethod) {
+			waitForSingleHotPlugIfaceOnVMISpec(hotPluggedVMI)
+			hotPluggedVMI = verifyDynamicInterfaceChange(hotPluggedVMI, plugMethod)
+
+			hotPluggedVM, err := kubevirt.Client().VirtualMachine(hotPluggedVM.Namespace).Get(context.Background(), hotPluggedVM.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			Eventually(func(g Gomega) {
```

Comment:
Q: By having this signature `func(g Gomega)`, it means the assertions in the body will not immediately fail without retry?